### ISSUE TITLE
Possible problem with draft and the publisher page

### ISSUE THREAD

**Issue by sethaj** [mlibrary/heliotrope#1286]
1. Make or import a new monograph but make it draft/private. It should include an epub.
2. When not logged in, go to that publisher's page (press catalog page, so /{publisher}).
3. Does the monograph show on that page? (Should it? We need to decide this. Probably not?)
4. When not logged in, can you get to this monograph's monograph_catalog page (ie: /concern/monographs/{NOID}). If it's draft and you're not logged in you shouldn't be able to get to this page (right?).

**Comment by sethaj** [comment]
When you add a FileSet to a Public monograph, does it turn it draft/private?

**Comment by conorom** [comment]
4. Yes, we have no restriction in `monograph_catalog_controller.rb` right now, so you can always get in and look at this page, even if the monograph is draft and you are logged out. You will only see "open" files in results though.

It is easy to put a check in the `load_presenter`method of this controller. I think that might be the place for it as this doesn't seem to be a "normal" Rails controller.

Your final comment is also correct, in a sense. Actually the UI makes this a little confusing. The permissions sidebar is meant to pertain only to the *Work*, but any new files being deposited in the edit are brought in with this potentially-new permission setting as well. You are only shown the "will I change existing files for you" screen if the permission setting is different from what the *Work* had before.

To put it another way: There doesn't seem to be any way (in the UI) to add a draft FileSet to an open Work (or vice versa) without setting that Work to draft also. You would then have to go back to the Work edit page and set the Work back to open, refusing the offer of changing the permissions of all files within the work on the "Apply changes to contents?" page:
`/concern/permissions/999999999/confirm_access`
